tests/e2e/es-index-cleaner/02-generate-spans.yaml

```diff
@@ -0,0 +1,27 @@
+# Generate some traces in the Jaeger instance
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: report-span-1
+spec:
+  template:
+    spec:
+      containers:
+        - name: report-span-1
+          image: local/asserts:e2e
+          command: ["./reporter",
```

Comment:
nit: I think is more readable to have this in another line, something like this:
```
[
   "./reporter",
   "--days", "5",
   "--services", "1",
    "--verbose"
]
```